WalletWasabi.Backend/Global.cs

```diff
@@ -113,10 +113,7 @@ public async Task InitializeAsync(CoordinatorRoundConfig roundConfig, Cancellati
 				{
 					throw new ArgumentException($"Risk indicators were not provided in {nameof(WabiSabiConfig)}.");
 				}
-
-				HttpClient.BaseAddress = url;
-
-				var coinVerifierApiClient = new CoinVerifierApiClient(CoordinatorParameters.RuntimeCoordinatorConfig.CoinVerifierApiAuthToken, RpcClient.Network, HttpClient);
+				var coinVerifierApiClient = new CoinVerifierApiClient(CreateHttpClient(), url, wabiSabiConfig.CoinVerifierApiAuthToken);
```

Comment:
Ultra nit:
```suggestion
				var coinVerifierApiClient = new CoinVerifierApiClient(CreateHttpClient(nameof(CoinVerifierApiClient)), url, wabiSabiConfig.CoinVerifierApiAuthToken);
```

or just simply `"CoinVerifier"`?